### ISSUE TITLE
Offer metrics for routers, not just for services and entrypoints

### ISSUE THREAD

**Issue by der-eismann** [traefik/traefik#7007]
### Do you want to request a *feature* or report a *bug*?

Feature

### What did you expect to see?

We recently upgraded from traefik 1.7 to 2.2 and while we had no problems with the migration to routers & middlewares we are lacking some metrics for the routers. We just have one entrypoint, so that doesn't really provide useful metrics. We want to know how many requests come per domain in kubernetes. It's quite a hassle to somehow link the service name with the actual domain/URL and we now have to rebuild all of our dashboards.

TL;DR: Router metrics would be cool, in addition to service and entrypoint metrics.

**Comment by notsureifkevin** [comment]
Related Issues: 

https://github.com/containous/traefik/issues/6495 (potential duplicate)

**Comment by der-eismann** [comment]
Yeah, I saw the issue, but I'm not sure if that's a duplicate. In his comment he clarifies:
> The metrics counts the traffic for each service, only get the number of requests for a service, but I want to get the network traffic for each service, such as the number of bytes of upstream traffic each service and the number of bytes of downstream traffic.

So his issue is more about transferred bytes in addition to requests as a metric, not explicitly about router metrics.

**Comment by JokerQueue** [comment]
+1

**Comment by jorge07** [comment]
Related #7510
